Philips.CodeAnalysis.Common/LiteralHelper.cs

```diff
@@ -42,7 +42,7 @@ private bool IsConstantExpression(ExpressionSyntax expression, SemanticModel sem
 			}
 			else
 			{
-				if (expression is TypeOfExpressionSyntax typeOfExpression && typeOfExpression.Type is PredefinedTypeSyntax)
+				if (expression is TypeOfExpressionSyntax { Type: PredefinedTypeSyntax })
```

Comment:
I'm not sure I've seen the is-operator used with property patterns before. Cool.

Reply:
Indeed, C# is getting some syntax tweaks in conditions lately.